include/lsst/jointcal/SimpleAstrometryMapping.h

```diff
@@ -45,81 +44,50 @@ class SimpleAstrometryMapping : public AstrometryMapping {
             : toBeFit(toBeFit),
               transform(astrometryTransform.clone()),
               errorProp(transform),
-              lin(new AstrometryTransformLinear) {
-        // in this order:
-        // take a copy of the input transform,
-        // assign the transformation used to propagate errors to the transform itself
-        // reserve some memory space to compute the derivatives (efficiency).
-    }
+              lin(new AstrometryTransformLinear) {}
 
     /// No copy or move: there is only ever one instance of a given mapping (i.e.. per ccd+visit)
     SimpleAstrometryMapping(SimpleAstrometryMapping const &) = delete;
     SimpleAstrometryMapping(SimpleAstrometryMapping &&) = delete;
     SimpleAstrometryMapping &operator=(SimpleAstrometryMapping const &) = delete;
     SimpleAstrometryMapping &operator=(SimpleAstrometryMapping &&) = delete;
 
+    /// @copydoc AstrometryMapping::freezeErrorTransform
     virtual void freezeErrorTransform() {
```

Comment:
1. I didn't know doxygen could figure that out.
2. Looks like the thing this would have overridden was commented out in the parent class (inherited from simastrom). I'm not going to dig into that question on this ticket, I'll just remove the copydoc and leave it as it was.

Reply:
Hmm... and the `@copydoc` didn't give you an error for a nonexistent method?

Reply:
It did and I missed it until now. I might not have done a `scons doc` before pushing.